evaluate.py

```diff
@@ -26,6 +26,7 @@ def ffwd(data_in, paths_out, checkpoint_dir, device_t='/gpu:0', batch_size=4):
     batch_size = min(len(paths_out), batch_size)
     curr_num = 0
     soft_config = tf.ConfigProto(allow_soft_placement=True)
```

Comment:
In Python 3 map returns iterator, so I had to change map() to list(map()). Forgot about additional prints, they can be removed.